__tests__/cli/deploy/bundle/DeployBundle.handler.test.ts

```diff
@@ -81,9 +82,9 @@ describe("bundle Handler", () => {
         } catch (e) {
             err = e;
         }
-        expect(err).toBeDefined();
-        expect(err).toBeInstanceOf(ImperativeError);
-        expect(err.message).toContain("--name parameter is not a string");
+        expect(err).toBeUndefined();
+        expect(params.arguments.errorMsg).toBeDefined();
+        expect(params.arguments.errorMsg).toContain("--name parameter is not a string");
```

Comment:
Are these changes converting from throwing Errors to setting errorMsg to work around the change in Imperative behaviour?  I'm concerning it means anyone calling the API directly has to know to check the error message.  I think we should at least find out if this change was intended.

Reply:
Yeah, any exception thrown the handler now causes Zowe to inform the user that there's a bug in the handler and that they should contact the authors.